python/GafferSceneUI/SceneEditor.py

```diff
@@ -65,6 +65,16 @@ def __init__( self, topLevelWidget, scriptNode, **kw ) :
 
 		self.__parentingConnections = {}
 
+	def editScope( self ) :
```

Comment:
> though may be a little odd as not every SceneEditor has an "editScope" plug on its
Settings node...

I think this is fine. It's entirely plausible that any of those could end up supporting EditScopes in future. I think there's even a case to be made that `Editor.Settings` should provide an `editScope` plug for all editors, and then we just hide the widget for editors that don't use it. Not saying we should do this now, just that I'm not concerned...

Reply:
Having read through the rest of the PR, I really wouldn't mind if `Editor.Settings` provided an EditScope for everything - seems like we could simplify things by removing a few checks if we did that. It would also be in keeping with the way the View base class provides an EditScope. Up to you whether you want to take that on now though...

Reply:
> Up to you whether you want to take that on now though...

In favour of seeing a 1.5.1.0 release out soonish, I'll punt on it for now, but I totally agree that it's the right direction to head...